src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.NonNetFx.cs

```diff
@@ -31,6 +31,9 @@ public static string LibcRelease
         {
             get
             {
+#if TargetsWindows
+                return "glibc_not_found";
```

Comment:
When running on Windows with .NET Framework, we're returning ""; why when running on Windows on .NET Core are we returning "glibc_not_found"?